test/test_xla_dist.py

```diff
@@ -587,7 +587,7 @@ def test_unhealthy_pod_service_cluster(self):
             'state':
                 'READY',
             'health':
-                'UNHEALTHY_TENSORFLOW',
```

Comment:
test_xla_dist can be removed, `xla_dist` only works for xrt

Reply:
thanks for pointing, test in https://github.com/pytorch/xla/pull/5224

Reply:
since #5224 has passed all tests, will apply this change

Reply:
Thanks, updated